packages/meditrak-app/app/widgets/BlueButton.js

```diff
@@ -4,7 +4,7 @@
  */
 import React from 'react';
 import { StyleSheet, View } from 'react-native';
-import { Button } from '.';
```

Comment:
This was a stab in the dark when the build was breaking, but it worked. Seems something about the appcenter build doesn't like to import from the index.js of the same directory